system-int-test/test/cmr/system_int_test/ingest/subscription_processing_test.clj

```diff
@@ -41,14 +41,13 @@
 (defn- create-granule-and-index
   "A utility function to reduce common code in these tests, Create a test granule and then wait for it to be indexed."
   [provider collection granule-ur]
-  (let [granule (data-core/ingest provider
-                    (data-granule/granule-with-umm-spec-collection collection
-                                                                   (:concept-id collection)
-                                                                   {:granule-ur granule-ur
-                                                                    :access-value 1})
-                    {:token "mock-echo-system-token"})]
+  (let [concept-id (:concept-id collection)
```

Comment:
I am not sure this is more readable than what we have before.

Reply:
I find it more readable, if only because `result` is returned instead of `granule`, which signifies that there is ingest response info along with the granule.